aip/general/0203.md

```diff
@@ -16,22 +16,24 @@ behavior (for example, to optimize client library signatures).
 
 ## Guidance
 
-APIs **should** use the `google.api.field_behavior` annotation to describe
-well-understood field behavior, such as a field's being required, immutable, or
-output only:
+APIs use the `google.api.field_behavior` annotation to describe well-understood
+field behavior, such as a field being required or immutable.
 
 ```proto
 // The audio data to be recognized.
 RecognitionAudio audio = 2 [(google.api.field_behavior) = REQUIRED];
 ```
 
-Additionally, APIs **may** use the `OPTIONAL` value to describe none of the
-above. However, it is never mandatory to explicitly describe a field as
-optional.
+- APIs **must** use the `google.api.field_behavior` annotation on every field.
```

Comment:
nit:

> ...**must** apply the...annotation to every...

"apply" or "include" are *slightly* better words here, imho.

Reply:
ack. I'll use the "apply" term, I think it's a little more direct than include.